### DISCUSSION TITLE
Signal Missed/keeps disconnecting

### DISCUSSION THREAD

**Discussion by Jaymienicole1990** [NightscoutFoundation/xDrip#3647] [General]
My drip+ keeps disconnecting I think I've tried all troubleshooting what should I do?
Using a Google Pixel G6 on Metro 
![Screenshot_20240827-225238](https://github.com/user-attachments/assets/008b3299-53cd-45af-882f-9085515d5878)
![Screenshot_20240827-225101](https://github.com/user-attachments/assets/c9755590-8e11-47a8-a368-69168f993d3b)

**Comment by Navid200** [comment]
Please edit your first post and include the phone brand and model.

**Reply by Navid200** [reply]
Pixel G6?
Can you provide a link?  

Have you set Bluetooth battery optimization to unrestricted.   That setting is extremely important for Pixel.

**Reply by Navid200** [reply]
![Screenshot_20240831-083049](https://github.com/user-attachments/assets/b4618fe5-a6dc-4f20-9ea1-3af86cdb05d7)

**Comment by jamorham** [comment]
That sensor status looks weird to me.
